deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -928,17 +945,18 @@ def create_mocked_agent(name="centos8-agent", ip="127.0.0.1", register_ip="127.0
     return agent_id_str
 
 
-def create_simulated_agent(manager_address="localhost", os="debian10"):
+@retry(AttributeError, attempts=2, delay=2, delay_multiplier=1)
+def create_simulated_agent(manager_address="localhost", operating_system="debian10"):
```

Comment:
Just a comment here. Keep in mind that you need to ensure the signature of this function is changed across the codebase. Otherwise, it will affect to other tests.